package.json

```diff
@@ -8,20 +8,20 @@
   },
   "scripts": {
     "build": "npm run build:dist && npm run build:dist6",
-    "build:dist": "tsc --target es2017 --outDir dist",
-    "build:dist6": "tsc --target es2015 --outDir dist6",
-    "build:watch": "tsc --watch",
-    "clean": "rm -rf dist dist6",
+    "build:dist": "lb-tsc --target es2017 --outDir dist",
+    "build:dist6": "lb-tsc --target es2015 --outDir dist6",
+    "build:watch": "lb-tsc --watch",
+    "clean": "lb-clean dist dist6",
```

Comment:
I suppose this change is an improvement, although it makes me wonder why is it needed to specify `dist` and `dist6` arguments? I'd expect that `lb-clean` should take care of cleaning all artifacts produced by `lb-*` scripts. What is the benefit of `lb-clean` over calling `rimraf` (as a replacement of non-portable `rm -rf`)?

Reply:
(See https://www.npmjs.com/package/rimraf)

Reply:
Same here.